src/main/java/com/google/sps/data/Comment.java

```diff
@@ -1,8 +1,10 @@
 package com.google.sps.data;
 
 import java.util.Objects;
+import com.google.sps.data.Constants;
 
 public final class Comment{
+    private static final double COMMENT_TOXICITY_THRESHOLD = 0.8;  
```

Comment:
why 0.8 - maybe add a comment explaining the threshold choice?

Reply:
it's just something random for now and we can decide later